src/Avalonia.Themes.Default/TitleBar.xaml

```diff
@@ -0,0 +1,53 @@
+<Styles xmlns="https://github.com/avaloniaui" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
+  <Design.PreviewWith>
+    <Border>
+      <TitleBar Background="SkyBlue" Height="30" Width="300" Foreground="Black" />
+    </Border>
+  </Design.PreviewWith>
+  <Style Selector="TitleBar">
+    <Setter Property="Foreground" Value="{DynamicResource SystemControlForegroundBaseHighBrush}"/>    
+    <Setter Property="VerticalAlignment" Value="Top" />
+    <Setter Property="HorizontalAlignment" Value="Stretch" />
+    <Setter Property="Background" Value="Transparent" />
+    <Setter Property="Template">
+      <ControlTemplate>
+        <Panel HorizontalAlignment="{TemplateBinding HorizontalAlignment}" VerticalAlignment="Stretch">
+          <Panel x:Name="PART_MouseTracker" Height="1" VerticalAlignment="Top" />
```

Comment:
I can't see `PART_MouseTracker` referred to in code, is this still needed? Or `PART_Container` or `PART_Background` for that matter.

Reply:
```
<Style Selector="TitleBar:fullscreen /template/ Panel#PART_MouseTracker">
    <Setter Property="Background" Value="Transparent" />
  </Style>
```

it works by becoming transparent in fullscreen mode... so it means the titlebar gets a pointerover applied...

that then triggers the animation to reveal the titlebar.

Reply:
Styles are referencing the other PARTs also.

Reply:
The `PART_` convention is usually used to mean that a control will be used in the code for the control. I can't actually find an explicit mention of this in the WPF docs, but I can find a few mentions on other sites such as http://bursjootech.blogspot.com/2008/11/naming-and-conventions-in-xaml.html

> "In WPF there is a convention for TemplateElements in controls that distinguishes controls that must exist in a template from other controls. This means that if I edit a control template I know that all elements named PART_* must be in the template and all other parts I can remove or do whatever I like to.

Because these parts aren't part of the contract between the code of the control and the template, they probably shouldn't have the `PART_` prefix. It's not a big deal I guess, but when I saw the `PART_` prefix and no code to get these controls I assumed it was dead code.